[RHACS] [Docs] ROX-30752: Docs for internal CA rotation (#99814)
openshift/openshift-docs: configuration/internal-certificate-authority-rotation-for-rhacs.adoc

:_mod-docs-content-type: ASSEMBLY
[id="internal-certificate-authority-rotation-for-rhacs"]
= Internal certificate authority rotation for RHACS
include::modules/common-attributes.adoc[]
:context: internal-certificate-authority-rotation-for-rhacs

toc::[]

[role="_abstract"]

The following points explain why certificate authority (CA) rotation is important in {rh-rhacs-first}:

> * {product-title-short} includes an internal CA with a 5-year validity. Central and secured cluster service certificates depend on this CA. If the CA expires, {product-title-short} stops working.

Reviewer: ```suggestion
* {product-title-short} includes an internal CA that is valid for 5 years. Central and secured cluster service certificates depend on this CA. If the CA expires, {product-title-short} stops working.
```
